### DISCUSSION TITLE
unable to deploy discord bot

### DISCUSSION THREAD

**Discussion by dimivelev** [vercel/vercel#4247] [General] [answered]
So i decided to deploy discord bot to zeit,but when i deployed it in logs it says:

BRANCH
No repository
Learn how to deploy via Git
Build Logs
10:22:22.513
Retrieving list of deployment files...
10:22:22.706
Downloading 4 deployment files...
10:22:24.097
Installing build runtime...
10:22:26.350
Build runtime installed: 2252.787ms
10:22:26.677
Looking up build cache...
10:22:26.875
Build cache found. Downloading...
10:22:27.117
Build cache downloaded [13.94 MB]: 241.791ms
10:22:27.449
downloading user files...
10:22:27.449
installing dependencies for user's code...
10:22:29.067

@discordjs/uws@10.149.0 install /zeit/26afec6e/node_modules/@discordjs/uws
10:22:29.067
node-gyp rebuild > build_log.txt 2>&1 || exit 0
10:22:29.867
npm WARN discord-bot@1.0.0 No repository field.
10:22:29.867
added 1 package from 2 contributors in 2.036s
10:22:29.900
running user script...
10:22:29.901
preparing lambda files...
10:22:29.901
compiling entrypoint with ncc...
10:22:30.109
ncc: Version 0.18.5
10:22:30.109
ncc: Compiling file index.js
10:22:31.533
ncc: Module directory "/zeit/26afec6e/node_modules/discord.js/src/client/websocket" attempted to require "erlpack" but could not be resolved, assuming external.
10:22:31.855
ncc: Module directory "/zeit/26afec6e/node_modules/discord.js/src/client/voice/opus" attempted to require "@discordjs/opus" but could not be resolved, assuming external.
10:22:31.870
ncc: Module directory "/zeit/26afec6e/node_modules/discord.js/src/client/voice/opus" attempted to require "node-opus" but could not be resolved, assuming external.
10:22:31.873
ncc: Module directory "/zeit/26afec6e/node_modules/discord.js/src/client/voice/opus" attempted to require "opusscript" but could not be resolved, assuming external.
10:22:32.344
ncc: Module directory "/zeit/26afec6e/node_modules/prism-media/src/transcoders/ffmpeg" attempted to require "ffmpeg-binaries" but could not be resolved, assuming external.
10:22:32.344
ncc: Module directory "/zeit/26afec6e/node_modules/prism-media/src/transcoders/ffmpeg" attempted to require "ffmpeg-static" but could not be resolved, assuming external.
10:22:32.354
ncc: Module directory "/zeit/26afec6e/node_modules/ws/lib" attempted to require "utf-8-validate" but could not be resolved, assuming external.
10:22:32.354
ncc: Module directory "/zeit/26afec6e/node_modules/ws/lib" attempted to require "bufferutil" but could not be resolved, assuming external.
10:22:32.518
ncc: Module directory "/zeit/26afec6e/node_modules/follow-redirects/node_modules/debug/src" attempted to require "supports-color" but could not be resolved, assuming external.
10:22:34.374
Uploading build outputs...
10:22:36.635
Build completed. Populating build cache...
10:22:37.892
Uploading build cache [15.41 MB]...
10:22:38.159
Build cache uploaded: 267.679ms
10:22:38.167
Done with "index.js"
You have reached the end of the logs.

**Comment by NotNite** [comment, marked as answer]
You probably don't want to be deploying a Discord bot here, as it's more designed for web apps. Maybe [Glitch](https://glitch.com) would be something better for your usecase?

**Reply by king101-bit** [reply]
How about heroku?
